index.html

```diff
@@ -272,50 +272,50 @@ <h3>RevocationList2020Status</h3>
 Verifiable Credentials Data Model specification [[VC-DATA-MODEL]]. The
 value is expected to be a URL that identifies the status information associated
 with the <a>verifiable credential</a>. It MUST NOT be the URL for the
-revocation list.
+status list.
               </td>
             </tr>
             <tr>
               <td>type</td>
               <td>
-The <code>type</code> property MUST be <code>RevocationList2020Status</code>.
+The <code>type</code> property MUST be <code>RevocationList2021</code>.
               </td>
             </tr>
             <tr>
-              <td>revocationListIndex</td>
+              <td>statusListIndex</td>
               <td>
-The <code>revocationListIndex</code> property MUST be an arbitrary size integer
+The <code>statusListIndex</code> property MUST be an arbitrary size integer
 greater than or equal to 0, expressed as a string. The value identifies the bit
 position of the revocation status of the <a>verifiable credential</a>.
               </td>
             </tr>
             <tr>
-              <td>revocationListCredential</td>
+              <td>statusListCredential</td>
               <td>
-The <code>revocationListCredential</code> property MUST be a URL to a
+The <code>statusListCredential</code> property MUST be a URL to a
 <a>verifiable credential</a>. When the URL is dereferenced, the resulting
 <a>verifiable credential</a> MUST have <code>type</code> property that
-includes the <code>RevocationList2020Credential</code> value.
+includes the <code>StatusList2021Credential</code> value.
               </td>
             </tr>
           </tbody>
         </table>
 
-        <pre class="example nohighlight" title="Example RevocationList2020Status">
+        <pre class="example nohighlight" title="Example RevocationList2021">
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/vc-revocation-list-2020/v1"
+    "https://w3id.org/vc-status-list-2021/v1"
   ],
   "id": "https://example.com/credentials/23894672394",
   "type": ["VerifiableCredential"],
   "issuer": "did:example:12345",
-  "issued": "2020-04-05T14:27:42Z",
+  "issued": "2021-04-05T14:27:42Z",
   <span class="highlight">"credentialStatus": {
     "id": "https://dmv.example.gov/credentials/status/3#94567",
-    "type": "RevocationList2020Status",
-    "revocationListIndex": "94567",
-    "revocationListCredential": "https://example.com/credentials/status/3"
+    "type": "RevocationList2021",
```

Comment:
This is not the list itself, but information about a particular status *in* the list.

```suggestion
    "type": "RevocationList2021Status",
```

There will need to be minor fixes around this throughout the spec.

Reply:
@tplooker please comment on this suggestion.

Reply:
@OR13 not sure I follow, sorry only just saw this comment, I agree with the suggested change made by @dlongley here.